Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apt-get update && apt-get install -qy software-properties-common python-soft
   apt-get clean && rm -rf /var/lib/apt/lists/*
 
 RUN wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py
+RUN npm install -g npm
```

Comment:
Don't do this like that. It needlessly increases size of the image, not to mention the fact that it gives you a "random" version of npm each time you build the image. If you want to change npm, update base image.